README.md

```diff
@@ -21,11 +21,12 @@ pip install framegrab
 ```
 
 ## Optional Dependencies
-To use a Basler USB or GigE camera, you must separately install the `pypylon` package.
+Certain camera types have additional dependencies that must be installed separately. If you don't intend to use these camera types, you don't need to install these extra packages. 
 
-Similarly, to use Intel RealSense cameras, you must install `pyrealsense2`. 
+- To use a Basler USB or GigE camera, you must separately install the `pypylon` package.
+- To use Intel RealSense cameras, you must install `pyrealsense2`.
+- To use a CSI2 camera connected with a ribbon cable to a Raspberry Pi, you must install `python3-picamera2`. See install instructions at the [picamera2 github repository](https://github.com/raspberrypi/picamera2).
```

Comment:
```suggestion
- To use a Raspberry PI "CSI2" camera (connected with a ribbon cable), you must install the `picamera2` library.  See [pip](https://pypi.org/project/picamera2/) or the [picamera2 github repository](https://github.com/raspberrypi/picamera2).
```

Reply:
I specified that github repository because pip is not the recommended way to install `picamera2`. In order for it to work for me, I had to install it with apt (`sudo apt install -y python3-picamera2 --no-install-recommends`) and update the venv creation line in `groundlight-pi-gen` to include `--system-site-packages`